sw/device/silicon_creator/rom/e2e/BUILD

```
@@ -446,6 +446,72 @@ test_suite(
     tests = [":e2e_bootstrap_entry_{}".format(lc_state.lower()) for lc_state in structs.to_dict(CONST.LCV)],
 )
 
+otp_json(
+    name = "otp_json_bootstrap_rma",
+    partitions = [
+        otp_partition(
+            name = "CREATOR_SW_CFG",
+            items = {
+                "CREATOR_SW_CFG_RMA_SPIN_EN": "0x739",  # HARDENED_BOOL_TRUE
+                "CREATOR_SW_CFG_RMA_SPIN_CYCLES": "10",
+            },
+        ),
+    ],
+    visibility = ["//visibility:private"],
+)
+
+# This OTP image is tightly coupled with the "rom_bootstrap_rma" testpoint.
+# Despite its name ending with "rma", it actually puts the device into the PROD
+# lifecycle state.
+otp_image(
+    name = "otp_img_bootstrap_rma",
+    src = "//hw/ip/otp_ctrl/data:otp_json_prod",
+    overlays = STD_OTP_OVERLAYS + [
+        ":otp_json_bootstrap_rma",
+    ],
+    visibility = ["//visibility:private"],
+)
+
+bitstream_splice(
+    name = "rom_otp_bootstrap_rma",
```

nit (I think all bitstreams in this file use the production rom):
```suggestion
    name = "bitstream_bootstrap_rma",
```